featurebyte/feast/patch.py

```diff
@@ -58,18 +66,44 @@ def augment_response_with_on_demand_transforms(
             )
 
     initial_response = OnlineResponse(online_features_response)
-    initial_response_df = initial_response.to_df(include_event_timestamps=True)
+    initial_response_arrow: Optional[pyarrow.Table] = None
+    initial_response_dict: Optional[Dict[str, List[Any]]] = None
 
     # Apply on demand transformations and augment the result rows
     odfv_result_names = set()
     for odfv_name, _feature_refs in odfv_feature_refs.items():
         odfv = requested_odfv_map[odfv_name]
-        transformed_features_df = odfv.get_transformed_features_df(
-            initial_response_df,
-            full_feature_names,
+        if odfv.mode == "python":
```

Comment:
In our usage, what will the odfv mode be?

Reply:
`pandas` mode: https://github.com/feast-dev/feast/blob/master/sdk/python/feast/on_demand_feature_view.py#L92